<ui version="4.0" >
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>845</width>
    <height>770</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>Horux Simul</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" >
    <item row="0" column="0" >
     <widget class="QTreeWidget" name="treeWidget" >
      <property name="maximumSize" >
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <column>
       <property name="text" >
        <string>Site</string>
       </property>
      </column>
      <column>
       <property name="text" >
        <string>ID</string>
       </property>
      </column>
     </widget>
    </item>
    <item row="0" column="1" >
     <widget class="QTabWidget" name="tabWidget" >
      <property name="minimumSize" >
       <size>
        <width>590</width>
        <height>0</height>
       </size>
      </property>
      <property name="currentIndex" >
       <number>0</number>
      </property>
      <widget class="QWidget" name="device" >
       <attribute name="title" >
        <string>Device</string>
       </attribute>
       <widget class="QWidget" native="1" name="widget" >
        <property name="geometry" >
         <rect>
          <x>0</x>
          <y>0</y>
          <width>581</width>
          <height>601</height>
         </rect>
        </property>
        <property name="minimumSize" >
         <size>
          <width>0</width>
          <height>0</height>
         </size>
        </property>
        <widget class="QWidget" name="gridLayout" >
         <property name="geometry" >
          <rect>
           <x>10</x>
           <y>10</y>
           <width>571</width>
           <height>591</height>
          </rect>
         </property>
         <layout class="QGridLayout" />
        </widget>
       </widget>
      </widget>
      <widget class="QWidget" name="log" >
       <attribute name="title" >
        <string>log</string>
       </attribute>
       <layout class="QGridLayout" >
        <item row="0" column="0" colspan="3" >
         <widget class="QTextEdit" name="logText" >
          <property name="horizontalScrollBarPolicy" >
           <enum>Qt::ScrollBarAsNeeded</enum>
          </property>
          <property name="lineWrapMode" >
           <enum>QTextEdit::NoWrap</enum>
          </property>
          <property name="readOnly" >
           <bool>true</bool>
          </property>
          <property name="textInteractionFlags" >
           <set>Qt::NoTextInteraction</set>
          </property>
         </widget>
        </item>
        <item row="1" column="0" >
         <widget class="QPushButton" name="clearLog" >
          <property name="text" >
           <string>Clear</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1" >
         <widget class="QLabel" name="label" >
          <property name="text" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600; color:#0000ff;">Receive from Horux&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
        </item>
        <item row="1" column="2" >
         <widget class="QLabel" name="label_2" >
          <property name="text" >
           <string>&lt;html>&lt;head>&lt;meta name="qrichtext" content="1" />&lt;style type="text/css">
p, li { white-space: pre-wrap; }
&lt;/style>&lt;/head>&lt;body style=" font-family:'Sans Serif'; font-size:9pt; font-weight:400; font-style:normal;">
&lt;p style=" margin-top:0px; margin-bottom:0px; margin-left:0px; margin-right:0px; -qt-block-indent:0; text-indent:0px;">&lt;span style=" font-weight:600; color:#008000;">Send to Horux&lt;/span>&lt;/p>&lt;/body>&lt;/html></string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>845</width>
     <height>34</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuHorux" >
    <property name="title" >
     <string>Horux</string>
    </property>
    <addaction name="actionNew_site" />
    <addaction name="actionOpen_site" />
    <addaction name="actionClose_site" />
    <addaction name="separator" />
    <addaction name="actionSave_site" />
    <addaction name="actionSave_as" />
    <addaction name="separator" />
    <addaction name="actionQuit" />
   </widget>
   <widget class="QMenu" name="menuHardware" >
    <property name="title" >
     <string>Hardware</string>
    </property>
    <addaction name="actionAdd" />
    <addaction name="actionDelete" />
   </widget>
   <widget class="QMenu" name="menuHelp" >
    <property name="title" >
     <string>Help</string>
    </property>
    <addaction name="actionAbout" />
   </widget>
   <widget class="QMenu" name="menuSystem" >
    <property name="title" >
     <string>System</string>
    </property>
    <addaction name="actionStart" />
    <addaction name="actionStop" />
    <addaction name="separator" />
   </widget>
   <addaction name="menuHorux" />
   <addaction name="menuHardware" />
   <addaction name="menuSystem" />
   <addaction name="menuHelp" />
  </widget>
  <widget class="QStatusBar" name="statusbar" />
  <widget class="QToolBar" name="toolBar" >
   <property name="windowTitle" >
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea" >
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak" >
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_site" />
   <addaction name="actionOpen_site" />
   <addaction name="actionSave_site" />
   <addaction name="separator" />
   <addaction name="actionAdd" />
   <addaction name="actionDelete" />
   <addaction name="separator" />
   <addaction name="actionStart" />
   <addaction name="actionStop" />
  </widget>
  <action name="actionNew_site" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/images/14_layer_newlayer.png</iconset>
   </property>
   <property name="text" >
    <string>New site...</string>
   </property>
  </action>
  <action name="actionOpen_site" >
   <property name="icon" >
    <iconset resource="application.qrc" >:/images/folder.png</iconset>
   </property>
   <property name="text" >
    <string>Open site...</string>
   </property>
  </action>
  <action name="actionClose_site" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Close site</string>
   </property>
  </action>
  <action name="actionSave_site" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="application.qrc" >:/images/filesave.png</iconset>
   </property>
   <property name="text" >
    <string>Save site</string>
   </property>
  </action>
  <action name="actionSave_as" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Save as...</string>
   </property>
  </action>
  <action name="actionQuit" >
   <property name="text" >
    <string>Quit</string>
   </property>
  </action>
  <action name="actionHorux_Simul_help" >
   <property name="text" >
    <string>Horux Simul help...</string>
   </property>
  </action>
  <action name="actionImport_site" >
   <property name="text" >
    <string>Import site...</string>
   </property>
  </action>
  <action name="actionAdd" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="application.qrc" >:/images/db_add.png</iconset>
   </property>
   <property name="text" >
    <string>Add...</string>
   </property>
  </action>
  <action name="actionDelete" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="application.qrc" >:/images/db_remove.png</iconset>
   </property>
   <property name="text" >
    <string>Delete</string>
   </property>
  </action>
  <action name="actionModify" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="text" >
    <string>Modify...</string>
   </property>
  </action>
  <action name="actionRun" >
   <property name="text" >
    <string>Start</string>
   </property>
  </action>
  <action name="actionStart" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="application.qrc" >:/images/player_play.png</iconset>
   </property>
   <property name="text" >
    <string>Power on</string>
   </property>
  </action>
  <action name="actionStop" >
   <property name="enabled" >
    <bool>false</bool>
   </property>
   <property name="icon" >
    <iconset resource="application.qrc" >:/images/player_stop.png</iconset>
   </property>
   <property name="text" >
    <string>Power off</string>
   </property>
  </action>
  <action name="actionAbout" >
   <property name="text" >
    <string>About...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="application.qrc" />
 </resources>
 <connections>
  <connection>
   <sender>clearLog</sender>
   <signal>clicked()</signal>
   <receiver>logText</receiver>
   <slot>clear()</slot>
   <hints>
    <hint type="sourcelabel" >
     <x>254</x>
     <y>661</y>
    </hint>
    <hint type="destinationlabel" >
     <x>493</x>
     <y>363</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
